masterlist.yaml

```diff
@@ -394,43 +400,61 @@ plugins:
   - name: 'DLCRobot.esm'
     group: *dlcGroup
     dirty:
-      - <<: *dirtyPlugin
-        crc: 0xD69027EA
-        util: 'FO4Edit v3.2.33'
-        itm: 45
+      - <<: *quickClean
+        crc: 0xD69027EA # v1.10.114.0
+        util: 'FO4Edit v3.2.84 EXPERIMENTAL'
+        itm: 49
         udr: 38
         nav: 1
+      - <<: *quickClean
+        crc: 0x1954F90B # v1.10.114.0
+        util: 'FO4Edit v3.2.84 EXPERIMENTAL'
+        itm: 1
+        nav: 1
+    clean:
+      - crc: 0xBD2A3927 # v1.10.114.0
+        util: 'FO4Edit v3.2.84 EXPERIMENTAL'
+        nav: 1
```

Comment:
I don't think it's right to show `dirty` plugins as `clean`, even if there's nothing also that users (in most cases) can/should do about it. They're still dirty and may cause CTDs etc.

I just noticed that this is for the DLCs. Hm. I'm still not overly fond about this. DLCs are still dirty, even if they're as clean as they're likely to get.